requirements.txt

```diff
@@ -28,7 +28,7 @@ typing_inspect==0.9.0
 build==1.2.2.post1
 ruamel.yaml==0.18.10
 setproctitle==1.3.5
-SQLAlchemy==2.0.38
```

Comment:
Why remove this? This is required for the DB layer to work.

Reply:
the dependency I added already installs sqlalchemy, I was worried there might be some mismatch with the versions of sqlalchemy

Reply:
I would like to keep the SQLAlchemy as a direct dependency as well. If we get a conflict, I prefer to get a warning instead of it just floating up